src/timeunit.ts

```diff
@@ -47,6 +78,15 @@ export const SINGLE_TIMEUNITS = [
   TimeUnit.MINUTES,
   TimeUnit.SECONDS,
   TimeUnit.MILLISECONDS,
+  TimeUnit.UTCYEAR,
```

Comment:
UTCYEAR will already contain YEAR in the name from `containsTimeUnit` method. 

So I doubt if you really need to add all of these values.